app/src/main/java/org/akvo/flow/util/ConstantUtil.java

```diff
@@ -133,6 +133,9 @@
     public static final String SURVEY_GROUP_EXTRA = "survey_group";
     public static final String SURVEYED_LOCALE_ID_EXTRA = "surveyed_locale_id";
     public static final String RECORD_ID_EXTRA = "record_id";
+    public static final String IMAGE_URL_EXTRA = "image_url";
+    public static final String FORM_TITLE_EXTRA = "title";
```

Comment:
I was a little confused with the name as I though it was referring to forms as we know them, i.e. questions.

Reply:
But aren't Surveys actually forms? I used the new name. So this is the name of the form, not the question. I can use the old naming which would make it SURVEY_TITLE_EXTRA

Reply:
Ignore the comment. that was my mixup. I just saw now in the issue that you were discussing whether to include the form names or something else. 👍